### ISSUE TITLE
Not all details should be required on custom service offerings

### ISSUE THREAD

**Issue by toonvd** [ngine-io/ansible-collection-cloudstack#111]
Some hosting providers want to force a fixed cpu_speed for example.
At the moment this results in being unable to deploy a new VM.

Ansible:
```
parameters are required together: cpu, cpu_speed, memory
```

Cloudstack:
```
The cpu speed of this offering id: XXX is not customizable. This is predefined in the template.
```

It seems this is a mismatch between CloudStack capabilities and this collection:

https://github.com/ngine-io/ansible-collection-cloudstack/blob/44f183c1eaec14d422216b3787dcbdd98ceabb36/plugins/modules/cs_instance.py#L1111

**Comment by resmo** [comment]
Hi @toonvd, can you share details which cloudstack version this is related to?

**Comment by savvii-runner** [comment]
Hi

```
"cloudstackversion": "4.16.1.1"
```

This has to do with `custom constrained` service offerings. Image below.

![image](https://user-images.githubusercontent.com/112495382/187619871-05aa6611-2cf3-4ee4-8005-c08746cf5358.png)

**Comment by toonvd** [comment]
Hi

`"cloudstackversion": "4.16.1.1"`

This has to do with custom constrained service offerings. Image below.
![image](https://user-images.githubusercontent.com/1227745/187620337-8903a2ce-4322-4c09-b2b3-6b4bbad1c0a2.png)

**Comment by resmo** [comment]
fixed by #114
